docs/releasenotes.md

```diff
@@ -433,7 +433,7 @@ Known Issues:
 - RNNs do not support fp16
 - Training with CNNs does not support fp16
 - Users may encounter a warning that their performance database is out of date. The performance database can be updated by setting the environment variable for just the initial run of an application: `MIOPEN_FIND_ENFORCE=search`
-For more information on the performance database, see: https://rocmsoftwareplatform.github.io/MIOpen/doc/html/perfdatabase.html#
+For more information on the performance database, see: https://ROCm.github.io/MIOpen/doc/html/perfdatabase.html#
```

Comment:
Use `https://rocm.docs.amd.com/projects/MIOpen/en/latest/perfdatabase.html`

Reply:
This is out of scope of this PR (the purpose is fix URLs broken due to renaming of the github organization), but I'll do that for you.

Reply:
Oh no, this link is related to the very old MIOpen release (2018) and I would better keep it as is.

Reply:
So I left it as is intentionally. We can even denote 2018 version of the .html (but not me please)